events/README.md

```diff
@@ -0,0 +1,416 @@
+# Events
+
+## Events object
+
+```
+{
+  'name': string,
+  'description': string,
+  'room_id':string,
+  'template_id': string,
+  'enabled': boolean,
+  'lock': boolean,
+  'region': <'in' | 'us' | 'eu' | 'auto'>,
+  'peers': [
+    "peer_id": string,
+    "peer_id": string,
+    "peer_id": string
```

Comment:
all of them are individual objects if I am not wrong. then should be wrapped up with the flower braces `{ }`.

Reply:
Yes, thank you. I have wrote it in a better format now.